<launch>

<node pkg="grasp_estimator" name="right_arm_planning_node" type="right_arm_planning_node" respawn="true" output="screen">
</node>
<node pkg="grasp_estimator" name="data_acquiring_node" type="data_acquiring_node" respawn="true" output="screen">
</node>
  

</launch>